app/models/post.rb

```diff
@@ -0,0 +1,21 @@
+class Post < ApplicationRecord
+  # Associations
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  # Attributes
+  attribute :title, :string
+  attribute :text, :text
+  attribute :comments_counter, :integer, default: 0
+  attribute :likes_counter, :integer, default: 0
+
+  # Methods
+  def update_user_posts_counter
+    author.update(posts_counter: author.posts.count)
```

Comment:
- [ ] This is supposed to be incremented but it is just counting. You can add plus 1 to it.
- [ ] I will also encourage you to add the after_save to it for the method to always be called after each save. It would be like `after_save :update_user_posts_counter`